x/accounts/defaults/lockup/lockup.go

```
@@ -416,6 +369,81 @@ func getStakingDenom(ctx context.Context) (string, error) {
+				if !errorsmod.IsOf(err, stakingtypes.ErrNoUnbondingDelegation) {
+					return true, err
+				}
+
+				skipIteration = true
+			}
+
+			found := false
+			// check if the entry is still exist in the unbonding entries
+			if !skipIteration {
+				for _, e := range entries {
+					if e.CompletionTime.Equal(entry.EndTime) && entry.CreationHeight == entry.CreationHeight {
```

golangci-lint (1.62.2)

dupSubExpr: suspicious identical LHS and RHS for `==` operator

(gocritic)